fast/project-templates/mongodb/project.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# yaml-language-server: $schema=../../../../../../cloud-foundation-fabric/modules/project-factory/schemas/project.schema.json
+
+# TODO: edit and uncomment the following line to create the project in a folder
+
+parent: shared
+name: prod-shared-mongodb-0
+services:
+  - compute.googleapis.com
+  - logging.googleapis.com
+  - monitoring.googleapis.com
+
+automation:
+  # TODO: edit the automation project and optionally edit resource names
+  project: foo-prod-shared-iac-0
+  service_accounts:
+    rw:
+      description: Read/write automation service account for MongoDB.
+  bucket:
+    description: Terraform state bucket for MongoDB.
+    iam:
+      roles/storage.objectAdmin:
+        - automation/rw
+      roles/storage.objectViewer:
+        - automation/rw
+iam:
+  roles/compute.admin:
+    - automation/rw
+  roles/servicedirectory.admin:
+    - automation/rw
+shared_vpc_service_config:
+  host_project: dev-spoke-0
+  network_users:
+    - automation/rw
```

Comment:
I'd comment out `parent`, `automation`, `iam`, and `shared_vpc_service_config`, as this is not necessary for this template

Reply:
It is to have it work in fast, which is the primary use case

Reply:
I edited the file a bit, should be clearer to understand now.